tools/container/toolkit/toolkit.go

```diff
@@ -32,6 +32,9 @@ import (
 
 	"github.com/NVIDIA/nvidia-container-toolkit/internal/config"
 	"github.com/NVIDIA/nvidia-container-toolkit/internal/system/nvdevices"
+	pkgconfig "github.com/NVIDIA/nvidia-container-toolkit/pkg/config"
```

Comment:
let's give this a more specific name. An alternative is to move the types that we're actually using to a more meaningful package name under `/pkg/config`.